editor-service/src/main/webapp/resources/js/require/main.js

```diff
@@ -1,18 +1,47 @@
 requirejs.config({
     baseUrl: 'resources/js/compiled',
     paths: {
-        //angular
-        angular: '../libs/angular/angular'
+        angular: '../libs/angular/angular',
+
+        jquery: '../libs/jquery/jquery',
+        jqueryui: '../libs/jquery/ui/jquery-ui',
+        jquerytree: '../libs/jquery/treeview/jquery.treeview',
```

Comment:
Seems like jquery.line is missing. It is impossible to link nodes and draw gestures using right click now.